### PR TITLE
Add sample with full path.

### PR DESCRIPTION
**Pull request by NN---**
It is not clear from documentation how full module path is to be used.
Added such sample.

- [X] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [X] Make sure your PR complies with the [writer's guide][2].
- [X] Review the diff carefully as sometimes this can reveal issues.
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/writers-guide/

### CODE REVIEW

File: src/content/plugins/provide-plugin.md [hunk 1]
```diff
@@ -24,11 +24,21 @@ new webpack.ProvidePlugin({
 })
 ```
 
+Search path is current folder and node_modules.
```
Comment (montogeek) [inline src/content/plugins/provide-plugin.md]: What does this mean?

**Comment by NN---** [comment]
It means that relative path "src/file" doesn't work and must use path.join(__dirname

**Comment by montogeek** [comment]
@NN--- Friendly ping

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**